co-authors-plus.php

```diff
@@ -947,24 +949,27 @@ function filter_count_user_posts( $count, $user_id ) {
 	/**
 	 * Checks to see if the current user can set authors or not
 	 */
-	function current_user_can_set_authors( $post = null ) {
+	function current_user_can_set_authors( $post = null, $is_api_request = false ) {
```

Comment:
In 271bce4720509aa8fe21abc37c438dcf813964cd, you mentioned that you added this `$is_api_request` parameter, but you didn't say why.  Why is it important/useful?